react/features/base/connection/actions.native.ts

```diff
@@ -253,7 +254,7 @@ export function connectionFailed(
  *     connection: JitsiConnection
  * }}
  */
-function _connectionWillConnect(connection) {
+function _connectionWillConnect(connection: Object) {
```

Comment:
why Object and not any?

Reply:
We prefer to avoid any. Object is not very good either but it's still better than any. (any could also mean null or undefined)